index.html

```diff
@@ -13,29 +15,29 @@
 
     <h1 id="title">Our Demo Survey form</h1>
     <p id="description">Thank you for taking your time reviewing and filling this form</p>
-    <form action="" id="survey-form">
-      <div class="required">
+    <form id="survey-form">
+      <DIV class="required">
         <label for="name" id="name-label">Name</label><input type="text" name="name" id="name" placeholder="Enter your name" required>
         <label for="email" id="email-label">Email</label><input type="email" name="email" id="email" placeholder="Enter your email" required>
         <label for="number" id="number-label">Number</label><input type="number" name="number" id="number" min="5" max="20" placeholder="Enter your number">
-      </div>
+      </DIV>
```

Comment:
- [ ] Please consider using lowercase for elements and attribute names. Use [this as reference](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)